source/github-helpers/parse-backticks.tsx

```diff
@@ -2,26 +2,22 @@ import React from 'dom-chef';
 
 const splittingRegex = /`(.*?)`/g;
 
-function splitTextReducer(fragment: DocumentFragment, text: string, index: number): DocumentFragment {
-	// Code is always in odd positions
-	if (index % 2 && text.length >= 1) {
-		// `span.sr-only` keeps the backticks copy-pastable but invisible
-		fragment.append(
-			<code className="rgh-parse-backticks">
-				<span className="sr-only">`</span>
-				{text.trim()}
-				<span className="sr-only">`</span>
-			</code>
-		);
-	} else if (text.length > 0) {
-		fragment.append(text);
+export default function parseBackticks(description: string): DocumentFragment {
+	const fragment = new DocumentFragment();
+	for (const [index, text] of description.split(splittingRegex).entries()) {
+		if (index % 2 && text.length >= 1) {
+			// `span.sr-only` keeps the backticks copy-pastable but invisible
+			fragment.append(
+				<code className="rgh-parse-backticks">
+					<span className="sr-only">`</span>
+					{text.trim()}
+					<span className="sr-only">`</span>
+				</code>
+			);
+		} else if (text.length > 0) {
+			fragment.append(text);
+		}
 	}
 
 	return fragment;
 }
-
-export default function parseBackticks(description: string): DocumentFragment {
-	return description
-		.split(splittingRegex)
-		.reduce(splitTextReducer, new DocumentFragment());
-}
```

Comment:
# Test

Look at this very discussion’s title or a PR list: https://github.com/sindresorhus/refined-github/pulls